db/ddl.sql

```diff
@@ -114,5 +114,7 @@ CREATE TABLE `paypal_express_responses` (
   `kb_account_id` varchar(255) DEFAULT NULL,
   `kb_tenant_id` varchar(255) DEFAULT NULL,
   PRIMARY KEY (`id`),
-  KEY `idx_paypal_express_responses_on_kb_payment_id_kb_tenant_id` (`kb_payment_id`, `kb_tenant_id`)
+  KEY `idx_paypal_express_responses_on_kb_payment_id_kb_tenant_id` (`kb_payment_id`, `kb_tenant_id`),
+  KEY `idx_paypal_express_responses_on_kb_account_id` (`kb_account_id`),
+  KEY `idx_paypal_express_responses_on_payer_email` (`payer_email`)
```

Comment:
Could you add a [migration](http://docs.killbill.io/0.16/database_migrations.html) file, so it's picked up by KPM?

See https://github.com/killbill/killbill-paypal-express-plugin/commit/2ae1ba72383e2459f5362ed34caadb642791fa5e for an example.